Library/OcMainLib/OpenCoreMisc.c

```diff
@@ -248,7 +247,7 @@ STATIC
 EFI_STATUS
 EFIAPI
 OcToolDescribeEntry (
```

Comment:
Is this method still needed? Is this needed? I have replaced the code to do with system tool icons with [this line](https://github.com/acidanthera/OpenCorePkg/pull/263/files#diff-7ea53dcd0ee7bb9a7c053fd83a44b3b6fc4833cc3a22dbd22cd4b75fe59b02f8L1378-R1378) (!). `.icns` was removed in the flavours commit. So all that remains in here is support for `.lbl`/`.l2x`. Does anyone use this?!

Reply:
Oh, right, it should then be unused, yes. Please remove that.

Reply:
Hrm, it is not that simple. For flavours we can group the icons, but for the names no decent grouping is really possible: we cannot guess the name of each tool. lbl support needs to stay I believe.

Reply:
What I mean is, there are many paths (including basically always, in all real cases, as far as I can see, for tools in the Tools directory) where there is no pre-generated label and so one is generated on the fly instead, in this code: https://github.com/MikeBeaton/OpenCorePkg/blob/toggle-sip/Platform/OpenCanopy/Views/BootPicker.c#L1427-L1459 

Am not sure the (unused?) support for pre-generated labels is needed, in this case?

Reply:
Well, hard to say. Now that the font support is much better, maybe indeed drop that. Ok.